documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -1,92 +1,583 @@
+```text
+$ kubectl create serviceaccount -n sample-weblogic-operator-ns sample-weblogic-operator-sa
+```
 
-```shell
-$ helm repo list
+```text
+$ cd /tmp/weblogic-kubernetes-operator
+$ helm install sample-weblogic-operator \
+  kubernetes/charts/weblogic-operator \
+  --namespace sample-weblogic-operator-ns \
+  --set image=ghcr.io/oracle/weblogic-kubernetes-operator:{{< latestVersion >}} \
+  --set serviceAccount=sample-weblogic-operator-sa \
+  --set "enableClusterRoleBinding=true" \
+  --set "domainNamespaceSelectionStrategy=LabelSelector" \
+  --set "domainNamespaceLabelSelector=weblogic-operator\=enabled" \
+  --wait
 ```
+
+Or, instead of using the previous `helm install` command,
+create a YAML file named `custom-values.yaml` with the following contents:
+
 ```
-NAME           URL
-weblogic-operator    https://oracle.github.io/weblogic-kubernetes-operator/charts
+image: "ghcr.io/oracle/weblogic-kubernetes-operator:{{< latestVersion >}}"
+serviceAcount: "sample-weblogic-operator-sa"
+enableClusterRoleBinding: true
+domainNamespaceSelectionStrategy: LabelSelector
+domainNamespaceLabelSelector: "weblogic-operator=enabled"
 ```
 
-Install the operator from the repository:
+and call:
 
-```shell
-$ helm install weblogic-operator weblogic-operator/weblogic-operator
+```text
+$ helm install weblogic-operator kubernetes/charts/weblogic-operator \
+  --namespace sample-weblogic-operator-ns \
+  --values custom-values.yaml \
+  --wait
 ```
 
-#### Upgrade the operator
+This creates a Helm release named `sample-weblogic-operator`
+in the `sample-weblogic-operator-ns` namespace,
+configures a deployment and supporting resources for the operator,
+and deploys the operator.
+
+You can verify the operator installation by examining the output from the `helm install` command.
+
+To check if the operator is deployed and running,
+see [Troubleshooting]({{<relref "/userguide/managing-operators/troubleshooting.md">}}).
+
+**Notes**:
+- We have not set the `kubernetesPlatform` in this example, but this may be required
+  for your environment. See [Determine the platform setting](#determine-the-platform-setting).
+- For more information on specifying the registry credentials when the operator image is stored in a private registry, see
+  [Customizing operator image name, pull secret, and private registry](#customizing-operator-image-name-pull-secret-and-private-registry).
+- For information about referencing a operator Helm chart in a remote Helm repository instead
+  of referencing a locally downloaded version in your local file system,
+  see [Set up operator Helm chart access](#set-up-operator-helm-chart-access).
+- Do not include a backslash (`\`) before the equals sign (`=`) in a domain namespace label selector
+  when specifying the selector in YAML.
+  A backslash (`\`) is only required when specifying the selector on the command line using `--set`,
+  as in the previous example.
+
+### Set up domain namespaces
```

Comment:
okay.

Reply:
Sorry, I just realized that I mentioned the wrong section!!
My suggestion was:  merge this section into "Choose a domain namespace selection strategy" section, because the two sections are both about domain namespaces.

Reply:
That's a different duplication, but it is still intentional.  This section is about what to do _after_ you install the operator (setup namespaces), where-as the 'choose a namespace' section is what do _before_ you install the operator.